ObjCVideoQuickstart/ViewController.m

```diff
@@ -142,24 +138,28 @@ - (void)startPreview {
 }
 
 - (void)flipCamera {
-    if (self.camera.source == TVIVideoCaptureSourceFrontCamera) {
-        [self.camera selectSource:TVIVideoCaptureSourceBackCameraWide];
+    if (self.camera.source == TVICameraCaptureSourceFrontCamera) {
+        [self.camera selectSource:TVICameraCaptureSourceBackCameraWide];
     } else {
-        [self.camera selectSource:TVIVideoCaptureSourceFrontCamera];
+        [self.camera selectSource:TVICameraCaptureSourceFrontCamera];
     }
 }
 
 - (void)prepareLocalMedia {
     
-    // We will offer local audio and video when we connect to room.
+    // We will share local audio and video when we connect to room.
     
-    // Adding local audio track to localMedia
+    // Create an audio track.
     if (!self.localAudioTrack) {
-        self.localAudioTrack = [self.localMedia addAudioTrack:YES];
+        self.localAudioTrack = [TVILocalAudioTrack track];
+
+        if (!self.localAudioTrack) {
+            [self logMessage:@"Failed to add audio track"];
+        }
```

Comment:
localAudioTrack is of no use when client is not connected to a Room. Should we demonstrate deleting audioTrack in didDisconnect callback?